src/store/actions/streamsAction.js

```diff
@@ -25,3 +32,23 @@ export const fetchStreams = () => async (dispatch) => {
     dispatch(fetchStreamsFailure(error));
   }
 };
+
+const addNewStreamRequest = () => ({ type: ADD_NEW_STREAM_REQUEST });
+const addNewStreamSuccess = (data) => ({ type: ADD_NEW_STREAM_SUCCESS, data });
+const addNewStreamFailure = (error) => ({ type: ADD_NEW_STREAM_FAILURE, error });
+
+export const addNewStream = (newStr) => async (dispatch) => {
+  dispatch(addNewStreamRequest());
+  try {
+    await axios.post('/streams', newStr).then((response) => dispatch(addNewStreamSuccess(response.data)));
+    NotificationManager.success('Направление успешно добавлено', 'Success');
+  } catch (error) {
+    if (error.response && error.response.data) {
+      dispatch(addNewStreamFailure(error.response));
+      NotificationManager.error(error.response.data, 'Post error!', 5000);
```

Comment:
Тоже на русском нужно
```suggestion
      NotificationManager.error(error.response.data, 'Ошибка отправки данных!', 5000);
```

Reply:
@OmegonAlpharius  перевел